spec/dummy/Procfile

```diff
@@ -1,3 +1,3 @@
 web: rails s
-worker: bundle exec sidekiq -q monitoring -q wrangler -q proxyconf -q billing -q flavors -q tags -q cloud
-clock: bundle exec clockwork app/clock.rb
\ No newline at end of file
+worker: bundle exec sidekiq -q monitoring -q wrangler -q proxyconf -q billing -q flavors -q tags -q cloud -q feedback
```

Comment:
We discussed about queues and decided to use `feedback` as queue for migration updates.